openstates/in/bills.py

```diff
@@ -45,198 +45,185 @@ class INBillScraper(BillScraper):
     # Can turn this on or off. There are thousands of subjects and it takes hours.
     SCRAPE_SUBJECTS = True
```

Comment:
This isn't used anymore, so you can chuck it or re-implement it. I didn't notice any slowness due to scraping subjects, so maybe the comment is just out-of-date.